satviz-consumer/src/main/java/edu/kit/satviz/consumer/cli/HeatmapColorsType.java

```diff
@@ -29,11 +31,24 @@ public HeatmapColors convert(ArgumentParser parser, Argument arg, String value)
     }
     String[] stringArray = value.split(":");
     HeatmapColors heatmapColors = new HeatmapColors();
-    heatmapColors.setFromColor(Integer.parseInt(stringArray[0].substring(1)));
-    heatmapColors.setToColor(Integer.parseInt(stringArray[1].substring(1)));
+    heatmapColors.setHotColor(intToColor(Integer.parseInt(stringArray[0].substring(1))));
+    heatmapColors.setColdColor(intToColor(Integer.parseInt(stringArray[1].substring(1))));
```

Comment:
Gleiches wie oben drüber
```suggestion
    heatmapColors.setHotColor(Color.web(stringArray[1]));
```